### ISSUE TITLE
Things to try

### ISSUE THREAD

**Issue by gchrupala** [gchrupala/peppa#1]
x Projection layers
x Pretrained wav2vec audio encoder
o Cyclic learning rate scheduler as in platalea
x Used BertAdam for now
x Attention based pooling/aggregation
